### PR TITLE
flake8 - restoring rules F401 and F811 about import statements and redefinitions

### PR DESCRIPTION
**Pull request by ndushay**
~~HOLD - creating PR to main to trigger build -- need to ensure corrections I'm making are ok~~

part of #394

Restoring flake8 rules, as we want them:
- F401 "Imported but unused"
- F811 "Redefinition of unused name" <-- possibly problematic

### CODE REVIEW

File: libsys_airflow/dags/vendor/purge_archive.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 import logging
 
 from airflow import DAG
-from airflow.operators.empty import EmptyOperator
```
Comment (ndushay) [inline libsys_airflow/dags/vendor/purge_archive.py:5]: unused

File: tests/airflow_client.py [hunk 2]
```diff
@@ -1,5 +1,3 @@
-import pathlib
```
Comment (ndushay) [inline tests/airflow_client.py:1]: unused

File: tests/apps/vendor_app/test_vendor_management_view.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 from pytest_mock_resources import create_sqlite_fixture, Rows
 
 from libsys_airflow.plugins.vendor.models import Vendor, VendorInterface
-from tests.airflow_client import test_airflow_client
+from tests.airflow_client import test_airflow_client  # noqa: F401
```
Comment (ndushay) [inline tests/apps/vendor_app/test_vendor_management_view.py:7]: these were easier to except inline

File: tests/test_connections.py [hunk 4]
```diff
@@ -94,6 +94,8 @@ def test_create_connection_ftp(mocker):
     )
 
 
+"""
+UNUSED per flake8 F811
```
Comment (ndushay) [inline tests/test_connections.py:98]: @justinlittman here is your duplicated method.  I'll leave it to you to remove it;  I wasn't sure if I should.

**Review by ndushay — COMMENTED** [review]
ready for review now!

**Review by jmartin-sul — APPROVED** [review]
lgtm, feel free to merge if your question for justin isn't a blocking one

**Pull request merged**